tunelo/flask.py

```diff
@@ -51,8 +52,25 @@ def create_app(test_config=None):
     app.after_request(_add_vary_x_auth_token_header)
 
     from .api import root
+
     app.register_blueprint(root.bp)
-    #app.register_blueprint(some_blueprint.bp, url_prefix="/v1/...")
+
+    from .api import list_channels
+
+    app.register_blueprint(list_channels.bp)
```

Comment:
Blueprints are intended to group similar endpoints into one unit. I think it would make sense to put all the channel endpoints in a single blueprint module personally. Then it would have two blueprints, one for the "root" endpoint (which just returns info about the service) and one for the channel endpoint(s).